Solutions/Authomize/Analytic Rules/AWS_role_with_admin_privileges.yaml

```diff
@@ -0,0 +1,61 @@
+id: 734c00a0-a95b-44dd-9b69-d926ed44256d
+name: AWS role with admin privileges
+kind: Scheduled
+description: The policy detect the creation of new AWS roles with administrative privileges.the policy configuration allows limiting the policy to specific accounts.
```

Comment:
Change to "privileges. The policy configuration"

Reply:
done

Reply:
This was applicable to all the queries where this typo exists.

Reply:
I went  through all and updated all typos i believe.